src/schema/schema.graphql

```diff
@@ -2718,6 +2718,7 @@ input HeaderCheckConditionInput {
 	header: String!
 	index: Int
 	negative: Boolean
+	type: String
```

Comment:
I don't see this parameter present in soap api https://files.zimbra.com/docs/soap_api/10.0.0/api-reference/zimbraMail/ModifyFilterRules.html
is this gap in documentation or we are doing something custom

Reply:
I have referred classic UI for ModifyFilterRulesRequest API. Type parameter is sent within addressBookTest in the condition when fourth filter drop down value in 'Address in' filter is 'Contact'. So have added type parameter. 
```
{
    "condition": "anyof",
    "addressBookTest": [
        {
            "header": "FROM",
            "type": "contacts"
        }
    ]
}
```
Just checked by removing type parameter, API call got successfully executed. 
I will be closing this branch.